### DISCUSSION TITLE
Github Sponsors

### DISCUSSION THREAD

**Discussion by azzamsa** [Orange-OpenSource/hurl#2883] [Ideas] [answered]
Hi, 👋🏽 

Any reason not to enable Github Sponsor for Hurl?

Thank you for making Hurl 🍜

**Comment by jcamiel** [comment, marked as answer]
Hi @azzamsa 

Thanks for your proposition!

Hurl is currently given by [Orange](https://www.orange.com) to the communauty, so we don't accept sponsors on Hurl repository. You can help us by spreading good words about Hurl, talking/bloging about it and help us to gain more stars, we'll appreciate it!
